README.md

```diff
@@ -875,11 +875,14 @@ app.use('/parse', parseServer.app); // (Optional) Mounts the REST API
 parseGraphQLServer.applyGraphQL(app); // Mounts the GraphQL API
 parseGraphQLServer.applyPlayground(app); // (Optional) Mounts the GraphQL Playground - do NOT use in Production
 
-app.listen(1337, function() {
-  console.log('REST API running on http://localhost:1337/parse');
-  console.log('GraphQL API running on http://localhost:1337/graphql');
-  console.log('GraphQL Playground running on http://localhost:1337/playground');
-});
+(async () => {
+  await parseServer.start();
+  app.listen(1337, function() {
+    console.log('REST API running on http://localhost:1337/parse');
+    console.log('GraphQL API running on http://localhost:1337/graphql');
+    console.log('GraphQL Playground running on http://localhost:1337/playground');
+  });
+})();
```

Comment:
Could you remove the IIFE? Node >=14.8 supports `await` at the top level and Parse Server 6 is supported only from Node 14.21.0, so we're good.